<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="0" name="pytest" skips="103" tests="158" time="19.500"><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="122" name="test_missing_params" time="0.0144457817078"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="129" name="test_get_token_refresh" time="0.0184941291809"><system-out>token_type access_token
</system-out></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="142" name="test_negative_get_token_none" time="0.013200044632"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="158" name="test_get_token_sa" time="0.0102820396423"><system-out>token_type access_token
</system-out></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="175" name="test_negative_get_token" time="0.0160911083221"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="188" name="test_get_json" time="0.0144948959351"><system-out>{&apos;key&apos;: &apos;value&apos;} None OCR_id
</system-out></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="221" name="test_get_retries_exceeded" time="0.0162858963013"><system-out>None Max retries exceeded with url: None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="203" name="test_get_retries" time="0.0189590454102"><system-out>{&apos;key&apos;: &apos;value&apos;} None OCR_id
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="238" name="test_empty_get_sent_bad_json" time="0.0166132450104"><system-out>None None None
</system-out></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="253" name="test_empty_get_sent_203" time="0.00928902626038"><system-out>{} None None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="268" name="test_negative_get_sent_203" time="0.00971817970276"><system-out>{&apos;message&apos;: &apos;error message&apos;} error message None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="283" name="test_negative_get_sent_300" time="0.00964212417603"><system-out>{} 300 None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="313" name="test_negative_get_raise_conn_exc" time="0.0111439228058"><system-out>None some exception None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="298" name="test_negative_get_raise_http_exc" time="0.0126397609711"><system-out>None some exception None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="343" name="test_has_feature_success_default" time="0.00798225402832"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="328" name="test_negative_get_raise_oserror_exc" time="0.0106377601624"><system-out>None some exception None
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="360" name="test_has_feature_success_user_false" time="0.0100979804993"><system-out>{&apos;feature_flags&apos;: {&apos;user_deprecation_warning&apos;: False}, &apos;refresh_token&apos;: &apos;ABCDEFGS&apos;}
</system-out></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="351" name="test_has_feature_success_user_true" time="0.00855708122253"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="380" name="test_has_feature_invalid_bool" time="0.00844693183899"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="370" name="test_has_feature_invalid_key" time="0.00691986083984"></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="414" name="test_negative_check_task_status" time="0.0199289321899"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="396" name="test_check_task_status" time="0.0149440765381"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="449" name="test_negative_wait_on_completion_failure" time="0.0141439437866"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="433" name="test_wait_on_completion" time="0.0121912956238"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="482" name="test_negative_wait_on_completion_timeout" time="0.014068365097"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp" file="module_utils/test_netapp.py" line="465" name="test_negative_wait_on_completion_error" time="0.0130109786987"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="100" name="test_set_parameters" time="0.00516295433044"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="92" name="test_cmp" time="0.00324487686157"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="119" name="test_compare_and_update_values" time="0.00311613082886"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="146" name="test_look_up_working_environment_by_name_in_list" time="0.00308680534363"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="107" name="test_cd_action" time="0.00403809547424"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="198" name="test_get_working_environment_details" time="0.0117220878601"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="133" name="test_get_working_environments_info" time="0.0227200984955"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="278" name="test_create_account" time="0.00332283973694"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="154" name="test_get_working_environment_details_by_name" time="0.0146629810333"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="301" name="test_get_account_info" time="0.0125479698181"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="214" name="test_get_working_environment_detail_for_snapmirror" time="0.0203013420105"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="333" name="test_get_accounts_info" time="0.0138490200043"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="347" name="test_set_api_root_path" time="0.00393581390381"></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="284" name="test_get_or_create_account" time="0.0129582881927"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="360" name="test_get_occm_agents_by_account" time="0.0124971866608"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="316" name="test_get_account_id" time="0.0113210678101"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="374" name="test_get_occm_agents_by_name" time="0.0128347873688"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="174" name="test_delete_cloudmanager_aggregate_pass" time="0.00237894058228"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:174: &lt;py._xmlgen.raw object at 0x562ce494bbd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="204" name="test_update_cloudmanager_aggregate_pass" time="0.0042200088501"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:204: &lt;py._xmlgen.raw object at 0x562ce4952190&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="396" name="test_get_agents_info" time="0.0135774612427"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="233" name="test_create_cloudmanager_aggregate_by_workingenv_name_pass" time="0.00336408615112"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:233: &lt;py._xmlgen.raw object at 0x562ce4952550&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="257" name="test_delete_cloudmanager_aggregate_by_workingenv_name_pass" time="0.00272583961487"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:257: &lt;py._xmlgen.raw object at 0x562ce4952890&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="415" name="test_get_active_agents_info" time="0.0188281536102"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aws_fsx.TestMyModule" file="modules/test_na_cloudmanager_aws_fsx.py" line="104" name="test_module_fail_when_required_args_missing" time="0.00222396850586"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aws_fsx.py:104: &lt;py._xmlgen.raw object at 0x562ce4952cd0&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="443" name="test_get_occm_agent_by_id" time="0.0121128559113"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aws_fsx.TestMyModule" file="modules/test_na_cloudmanager_aws_fsx.py" line="111" name="test_module_fail_when_required_args_present" time="0.00583386421204"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aws_fsx.py:111: &lt;py._xmlgen.raw object at 0x562ce4c5c250&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="463" name="test_check_occm_status" time="0.00936818122864"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aws_fsx.TestMyModule" file="modules/test_na_cloudmanager_aws_fsx.py" line="123" name="test_create_cloudmanager_aws_fsx_pass" time="0.00890898704529"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aws_fsx.py:123: &lt;py._xmlgen.raw object at 0x562ce49528d0&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="481" name="test_register_agent_to_service" time="0.011883020401"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="516" name="test_delete_occm" time="0.0121750831604"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_aws_fsx.TestMyModule" file="modules/test_na_cloudmanager_aws_fsx.py" line="147" name="test_delete_cloudmanager_aws_fsx_pass" time="0.00307297706604"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aws_fsx.py:147: &lt;py._xmlgen.raw object at 0x562ce493bf90&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="531" name="test_delete_occm_agents" time="0.0121002197266"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="module_utils.test_netapp_module" file="module_utils/test_netapp_module.py" line="555" name="test_get_tenant" time="0.0110287666321"><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="106" name="test_module_fail_when_required_args_missing" time="0.00351095199585"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:106: &lt;py._xmlgen.raw object at 0x562ce4c5cad0&gt;</skipped></testcase><testcase classname="module_utils.test_netapp_module_open" file="module_utils/test_netapp_module_open.py" line="47" name="test_certificates" time="0.00412702560425"><skipped message="skipping as builtins not defined for 2.6 and 2.7" type="pytest.skip">module_utils/test_netapp_module_open.py:47: &lt;py._xmlgen.raw object at 0x562ce4c5ce10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="139" name="test_module_fail_when_required_args_missing" time="0.00299620628357"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:139: &lt;py._xmlgen.raw object at 0x562ce4c640d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="146" name="test_module_fail_when_required_args_present" time="0.00321006774902"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:146: &lt;py._xmlgen.raw object at 0x562ce4c64310&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="130" name="test_create_cifs_server_idempotency" time="0.0106008052826"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:130: &lt;py._xmlgen.raw object at 0x562ce4c64510&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_aggregate.TestMyModule" file="modules/test_na_cloudmanager_aggregate.py" line="156" name="test_create_cloudmanager_aggregate_pass" time="0.00212502479553"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_aggregate.py:156: &lt;py._xmlgen.raw object at 0x562ce4c64790&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="357" name="test_create_instance_no_ami_with_tags" time="0.00304102897644"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:357: &lt;py._xmlgen.raw object at 0x562ce4c649d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="389" name="test_create_instance_timeout" time="0.00286221504211"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:389: &lt;py._xmlgen.raw object at 0x562ce4c64c10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="413" name="test_create_instance_error_in_get_agent" time="0.00214195251465"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:413: &lt;py._xmlgen.raw object at 0x562ce4c5ce10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="151" name="test_create_cifs_server_using_workgroup_successfully" time="0.0133030414581"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:151: &lt;py._xmlgen.raw object at 0x562ce4c5c8d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="168" name="test_delete_cifs_server_successfully" time="0.00531101226807"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:168: &lt;py._xmlgen.raw object at 0x562ce493bf10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="437" name="test_create_instance_error_in_get_account" time="0.0036141872406"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:437: &lt;py._xmlgen.raw object at 0x562ce494bbd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="460" name="test_create_instance_error_in_register" time="0.00238490104675"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:460: &lt;py._xmlgen.raw object at 0x562ce4952610&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="193" name="test_create_cifs_server_successfully" time="0.00388216972351"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:193: &lt;py._xmlgen.raw object at 0x562ce4c64cd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="485" name="test_create_instance_error_in_open" time="0.00233316421509"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:485: &lt;py._xmlgen.raw object at 0x562ce4c649d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="513" name="test_delete_instance" time="0.00207805633545"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:513: &lt;py._xmlgen.raw object at 0x562ce4c642d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cifs_server.TestMyModule" file="modules/test_na_cloudmanager_cifs_server.py" line="217" name="test_delete_cifs_server_with_workingenv_name_successfully" time="0.00428891181946"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cifs_server.py:217: &lt;py._xmlgen.raw object at 0x562ce4c640d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="532" name="test_delete_no_client" time="0.00319623947144"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:532: &lt;py._xmlgen.raw object at 0x562ce4c6f150&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="111" name="test_module_fail_when_required_args_missing" time="0.0035240650177"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:111: &lt;py._xmlgen.raw object at 0x562ce4c6f350&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="556" name="test_delete_instance_timeout" time="0.00205707550049"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:556: &lt;py._xmlgen.raw object at 0x562ce4c6f510&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="118" name="test_module_fail_when_required_args_present" time="0.00269412994385"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:118: &lt;py._xmlgen.raw object at 0x562ce4c6f790&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="572" name="test_delete_instance_error_on_agent" time="0.00209283828735"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:572: &lt;py._xmlgen.raw object at 0x562ce4c6f9d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="128" name="test_create_cloudmanager_connector_aws_pass" time="0.00273704528809"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:128: &lt;py._xmlgen.raw object at 0x562ce4c6fc10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="588" name="test_delete_instance_client_id_not_found_403" time="0.0057201385498"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:588: &lt;py._xmlgen.raw object at 0x562ce4c6fe50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="152" name="test_delete_cloudmanager_connector_aws_pass" time="0.00296401977539"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:152: &lt;py._xmlgen.raw object at 0x562ce4c64310&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="607" name="test_delete_instance_client_id_not_found_other" time="0.0151660442352"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:607: &lt;py._xmlgen.raw object at 0x562ce4c64c90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="177" name="test_delete_cloudmanager_connector_aws_pass_no_ids" time="0.00261402130127"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:177: &lt;py._xmlgen.raw object at 0x562ce4952610&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="625" name="test_delete_instance_account_id_not_found" time="0.00277996063232"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:625: &lt;py._xmlgen.raw object at 0x562ce493bf50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="212" name="test_delete_cloudmanager_connector_aws_negative_no_instance" time="0.00311994552612"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:212: &lt;py._xmlgen.raw object at 0x562ce4c5ce90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="646" name="test_modify_instance" time="0.00288510322571"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:646: &lt;py._xmlgen.raw object at 0x562ce4c6f990&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="246" name="test_get_instance_empty" time="0.00745296478271"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:246: &lt;py._xmlgen.raw object at 0x562ce4c6f5d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_azure" file="modules/test_na_cloudmanager_connector_azure.py" line="123" name="test_module_fail_when_required_args_missing" time="0.00241780281067"><skipped message="skipping as missing required imports on 2.6 and 2.7: No module named azure.mgmt.resource" type="pytest.skip">modules/test_na_cloudmanager_connector_azure.py:123: &lt;py._xmlgen.raw object at 0x562ce4c6f250&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="260" name="test_get_instance_one" time="0.00228095054626"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:260: &lt;py._xmlgen.raw object at 0x562ce4c7c150&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_azure" file="modules/test_na_cloudmanager_connector_azure.py" line="131" name="test_module_fail_when_required_args_present" time="0.00265693664551"><skipped message="skipping as missing required imports on 2.6 and 2.7: No module named azure.mgmt.resource" type="pytest.skip">modules/test_na_cloudmanager_connector_azure.py:131: &lt;py._xmlgen.raw object at 0x562ce4c7c3d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="274" name="test_get_instance_many_terminated" time="0.00955319404602"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:274: &lt;py._xmlgen.raw object at 0x562ce4c7c650&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_azure" file="modules/test_na_cloudmanager_connector_azure.py" line="142" name="test_create_cloudmanager_connector_azure_pass" time="0.00266909599304"><skipped message="skipping as missing required imports on 2.6 and 2.7: No module named azure.mgmt.resource" type="pytest.skip">modules/test_na_cloudmanager_connector_azure.py:142: &lt;py._xmlgen.raw object at 0x562ce4c7c810&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="306" name="test_get_instance_many_but_only_one_active" time="0.00257110595703"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:306: &lt;py._xmlgen.raw object at 0x562ce4c7cb50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_azure" file="modules/test_na_cloudmanager_connector_azure.py" line="161" name="test_delete_cloudmanager_connector_azure_pass" time="0.00239491462708"><skipped message="skipping as missing required imports on 2.6 and 2.7: No module named azure.mgmt.resource" type="pytest.skip">modules/test_na_cloudmanager_connector_azure.py:161: &lt;py._xmlgen.raw object at 0x562ce4c7cd10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="323" name="test_get_instance_exception" time="0.00258708000183"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:323: &lt;py._xmlgen.raw object at 0x562ce4c7ce10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="127" name="test_module_fail_when_required_args_missing" time="0.00217390060425"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:127: &lt;py._xmlgen.raw object at 0x562ce4c84190&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="135" name="test_module_fail_when_required_args_present" time="0.00213718414307"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:135: &lt;py._xmlgen.raw object at 0x562ce4c7cc10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_aws.TestMyModule" file="modules/test_na_cloudmanager_connector_aws.py" line="338" name="test_create_instance" time="0.00254392623901"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_connector_aws.py:338: &lt;py._xmlgen.raw object at 0x562ce4c7c710&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="367" name="test_delete_idempotent" time="0.00221705436707"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:367: &lt;py._xmlgen.raw object at 0x562ce4c7c250&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="148" name="test_create_cloudmanager_connector_gcp_pass" time="0.00286412239075"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:148: &lt;py._xmlgen.raw object at 0x562ce493bf50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="119" name="test_module_fail_when_required_args_missing" time="0.00331902503967"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:119: &lt;py._xmlgen.raw object at 0x562ce4c64bd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="174" name="test_delete_cloudmanager_connector_gcp_pass" time="0.0026741027832"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:174: &lt;py._xmlgen.raw object at 0x562ce4c6f210&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="126" name="test_module_fail_when_required_args_present" time="0.00348591804504"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:126: &lt;py._xmlgen.raw object at 0x562ce4c6f850&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="248" name="test_delete_occm_gcp_pass" time="0.00240731239319"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:248: &lt;py._xmlgen.raw object at 0x562ce4c6fe50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="270" name="test_create_occm_gcp_pass" time="0.00270080566406"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:270: &lt;py._xmlgen.raw object at 0x562ce4c843d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="136" name="test_create_cloudmanager_cvo_aws_pass" time="0.00227904319763"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:136: &lt;py._xmlgen.raw object at 0x562ce4c84590&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="292" name="test_get_deploy_vm_pass" time="0.00236392021179"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:292: &lt;py._xmlgen.raw object at 0x562ce4c84850&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="162" name="test_create_cloudmanager_cvo_aws_ha_pass" time="0.00255107879639"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:162: &lt;py._xmlgen.raw object at 0x562ce4c84b10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="310" name="test_get_occm_agents_absent_pass" time="0.00310206413269"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:310: &lt;py._xmlgen.raw object at 0x562ce4c84d50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="328" name="test_get_occm_agents_present_pass" time="0.00195693969727"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:328: &lt;py._xmlgen.raw object at 0x562ce4c8b050&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="193" name="test_create_cloudmanager_cvo_aws_capacity_license_pass" time="0.00695109367371"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:193: &lt;py._xmlgen.raw object at 0x562ce4c8b210&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_connector_gcp" file="modules/test_na_cloudmanager_connector_gcp.py" line="346" name="test_create_idempotent" time="0.00209212303162"><skipped message="skipping as missing required google packages on 2.6 and 2.7" type="pytest.skip">modules/test_na_cloudmanager_connector_gcp.py:346: &lt;py._xmlgen.raw object at 0x562ce4c84f90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="155" name="test_create_cloudmanager_cvo_azure_pass" time="0.00248599052429"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:155: &lt;py._xmlgen.raw object at 0x562ce4c84950&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="223" name="test_create_cloudmanager_cvo_aws_ha_capacity_license_pass" time="0.00856995582581"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:223: &lt;py._xmlgen.raw object at 0x562ce4c845d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="179" name="test_create_cloudmanager_cvo_azure_capacity_license_pass" time="0.00208210945129"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:179: &lt;py._xmlgen.raw object at 0x562ce4c84110&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="255" name="test_create_cloudmanager_cvo_aws_nodebase_license_pass" time="0.00557804107666"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:255: &lt;py._xmlgen.raw object at 0x562ce4c6f150&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="207" name="test_create_cloudmanager_cvo_azure_ha_capacity_license_pass" time="0.00458097457886"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:207: &lt;py._xmlgen.raw object at 0x562ce4952350&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="236" name="test_create_cloudmanager_cvo_azure_nodebase_license_pass" time="0.00264120101929"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:236: &lt;py._xmlgen.raw object at 0x562ce4c7c250&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="283" name="test_create_cloudmanager_cvo_aws_ha_nodebase_license_pass" time="0.00260591506958"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:283: &lt;py._xmlgen.raw object at 0x562ce4c7cb10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="314" name="test_delete_cloudmanager_cvo_aws_pass" time="0.0027277469635"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:314: &lt;py._xmlgen.raw object at 0x562ce4c8b050&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="262" name="test_create_cloudmanager_cvo_azure_ha_nodebase_license_pass" time="0.00313401222229"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:262: &lt;py._xmlgen.raw object at 0x562ce4c8b690&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_aws.TestMyModule" file="modules/test_na_cloudmanager_cvo_aws.py" line="336" name="test_change_cloudmanager_cvo_aws" time="0.00284767150879"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_aws.py:336: &lt;py._xmlgen.raw object at 0x562ce4c8b8d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="292" name="test_create_cloudmanager_cvo_azure_ha_pass" time="0.00284790992737"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:292: &lt;py._xmlgen.raw object at 0x562ce4c8bb10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="137" name="test_module_fail_when_required_args_missing" time="0.00271606445312"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:137: &lt;py._xmlgen.raw object at 0x562ce4c8bd50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="145" name="test_module_fail_when_required_args_present" time="0.00200819969177"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:145: &lt;py._xmlgen.raw object at 0x562ce4c8bf90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="171" name="test_create_cloudmanager_cvo_gcp_ha_pass" time="0.00273394584656"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:171: &lt;py._xmlgen.raw object at 0x562ce4c97210&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="320" name="test_delete_cloudmanager_cvo_azure_pass" time="0.00395917892456"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:320: &lt;py._xmlgen.raw object at 0x562ce4c97450&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="207" name="test_create_cloudmanager_cvo_gcp_capacity_license_pass" time="0.00316786766052"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:207: &lt;py._xmlgen.raw object at 0x562ce4c8bf90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_azure.TestMyModule" file="modules/test_na_cloudmanager_cvo_azure.py" line="342" name="test_change_cloudmanager_cvo_azure" time="0.00443887710571"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_azure.py:342: &lt;py._xmlgen.raw object at 0x562ce4c8b990&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="236" name="test_create_cloudmanager_cvo_gcp_ha_capacity_license_pass" time="0.00328803062439"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:236: &lt;py._xmlgen.raw object at 0x562ce4c8b710&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="110" name="test_module_fail_when_required_args_missing" time="0.00290393829346"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:110: &lt;py._xmlgen.raw object at 0x562ce4c7cd50&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="137" name="test_module_fail_when_required_args_present" time="0.00408387184143"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:137: &lt;py._xmlgen.raw object at 0x562ce49528d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="274" name="test_create_cloudmanager_cvo_gcp_nodebase_license_pass" time="0.00881195068359"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:274: &lt;py._xmlgen.raw object at 0x562ce4c64390&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="302" name="test_create_cloudmanager_cvo_gcp_ha_nodebase_license_pass" time="0.00294518470764"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:302: &lt;py._xmlgen.raw object at 0x562ce4c84590&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="340" name="test_delete_cloudmanager_cvo_gcp_pass" time="0.00240778923035"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:340: &lt;py._xmlgen.raw object at 0x562ce4c84d90&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="362" name="test_change_cloudmanager_cvo_gcp" time="0.00312685966492"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:362: &lt;py._xmlgen.raw object at 0x562ce4c97190&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="147" name="test_create_cloudmanager_cvo_gcp_pass" time="0.0111589431763"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:147: &lt;py._xmlgen.raw object at 0x562ce4c97690&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_cvo_gcp.TestMyModule" file="modules/test_na_cloudmanager_cvo_gcp.py" line="437" name="test_change_cloudmanager_cvo_gcp_ha" time="0.00240397453308"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_cvo_gcp.py:437: &lt;py._xmlgen.raw object at 0x562ce4c97910&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_info" file="modules/test_na_cloudmanager_info.py" line="477" name="test_get_accounts_info" time="0.014732837677"><system-out>Info: test_create_cloudmanager_info: AnsibleExitJson({&apos;info&apos;: {&apos;accounts_info&apos;: ({&apos;azureAccounts&apos;: [{&apos;vsaList&apos;: [{&apos;workingEnvironmentType&apos;: &apos;AZURE&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-jI0tbceH&apos;, &apos;name&apos;: &apos;testAZURE&apos;}, {&apos;workingEnvironmentType&apos;: &apos;AZURE&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-00EnDcfB&apos;, &apos;name&apos;: &apos;test&apos;}], &apos;publicId&apos;: &apos;CloudProviderAccount-T84ceMYu&apos;, &apos;tenantId&apos;: &apos;1&apos;, &apos;accountType&apos;: &apos;AZURE_KEYS&apos;, &apos;accountName&apos;: &apos;AzureKeys&apos;, &apos;applicationId&apos;: &apos;1&apos;}, {&apos;vsaList&apos;: [], &apos;publicId&apos;: &apos;CloudProviderAccount-XxbN95dj&apos;, &apos;tenantId&apos;: &apos;1&apos;, &apos;accountType&apos;: &apos;AZURE_KEYS&apos;, &apos;accountName&apos;: &apos;s&apos;, &apos;applicationId&apos;: &apos;1&apos;}], &apos;nssAccounts&apos;: [{&apos;vsaList&apos;: [{&apos;workingEnvironmentType&apos;: &apos;AWS&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-3txYJOsX&apos;, &apos;name&apos;: &apos;testAWS&apos;}, {&apos;workingEnvironmentType&apos;: &apos;AZURE&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-jI0tbceH&apos;, &apos;name&apos;: &apos;testAZURE&apos;}], &apos;nssUserName&apos;: &apos;TESTCLOUD2&apos;, &apos;publicId&apos;: &apos;be2f3cac-352a-46b9-a341-a446c35b61c9&apos;, &apos;accountType&apos;: &apos;NSS_KEYS&apos;, &apos;accountName&apos;: &apos;TESTCLOUD2&apos;}, {&apos;vsaList&apos;: [{&apos;workingEnvironmentType&apos;: &apos;AZURE&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-00EnDcfB&apos;, &apos;name&apos;: &apos;test&apos;}], &apos;nssUserName&apos;: &apos;ntapitdemo&apos;, &apos;publicId&apos;: &apos;01e43a7d-cfc9-4682-aa12-15374ce81638&apos;, &apos;accountType&apos;: &apos;NSS_KEYS&apos;, &apos;accountName&apos;: &apos;ntapitdemo&apos;}], &apos;awsAccounts&apos;: [{&apos;vsaList&apos;: [], &apos;publicId&apos;: &apos;CloudProviderAccount-Ekj6L9QX&apos;, &apos;accessKey&apos;: &apos;1&apos;, &apos;accountName&apos;: &apos;tami&apos;, &apos;accountType&apos;: &apos;AWS_KEYS&apos;, &apos;subscriptionId&apos;: &apos;hackExp10Days&apos;, &apos;accountId&apos;: &apos;123456789011&apos;}, {&apos;vsaList&apos;: [{&apos;workingEnvironmentType&apos;: &apos;AWS&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-9m3I6i3I&apos;, &apos;name&apos;: &apos;CVO_AWSCluster&apos;}, {&apos;workingEnvironmentType&apos;: &apos;AWS&apos;, &apos;publicId&apos;: &apos;VsaWorkingEnvironment-JCzkA9OX&apos;, &apos;name&apos;: &apos;testAWS1&apos;}], &apos;publicId&apos;: &apos;InstanceProfile&apos;, &apos;accountType&apos;: &apos;INSTANCE_PROFILE&apos;, &apos;accessKey&apos;: &apos;&apos;, &apos;accountName&apos;: &apos;Instance Profile&apos;, &apos;subscriptionId&apos;: &apos;hackExp10Days&apos;, &apos;occmRole&apos;: &apos;occmRole&apos;, &apos;accountId&apos;: &apos;123456789011&apos;}], &apos;gcpStorageAccounts&apos;: []}, None)}, &apos;changed&apos;: False},)
</system-out></testcase><testcase classname="modules.test_na_cloudmanager_info" file="modules/test_na_cloudmanager_info.py" line="100" name="test_module_fail_when_required_args_missing" time="0.0207889080048"><system-out>Info: missing required arguments: client_id
</system-out></testcase><testcase classname="modules.test_na_cloudmanager_nss_account.TestMyModule" file="modules/test_na_cloudmanager_nss_account.py" line="75" name="test_module_fail_when_required_args_missing" time="0.00241112709045"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_nss_account.py:75: &lt;py._xmlgen.raw object at 0x7feda0062410&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_info" file="modules/test_na_cloudmanager_info.py" line="108" name="test_get_working_environments_info" time="0.0195550918579"><system-out>Info: test_create_cloudmanager_info: AnsibleExitJson({&apos;info&apos;: {&apos;working_environments_info&apos;: ({&apos;vsaWorkingEnvironments&apos;: [{&apos;publicId&apos;: &apos;VsaWorkingEnvironment-aws12345&apos;, &apos;name&apos;: &apos;testAWS&apos;, &apos;tenantId&apos;: &apos;Tenant-2345&apos;, &apos;isHA&apos;: False, &apos;cloudProviderName&apos;: &apos;Amazon&apos;, &apos;workingEnvironmentType&apos;: &apos;VSA&apos;, &apos;creatorUserEmail&apos;: &apos;samlp|NetAppSAML|testuser&apos;}, {&apos;publicId&apos;: &apos;VsaWorkingEnvironment-awsha345&apos;, &apos;name&apos;: &apos;testAWSHA&apos;, &apos;tenantId&apos;: &apos;Tenant-2345&apos;, &apos;isHA&apos;: True, &apos;cloudProviderName&apos;: &apos;Amazon&apos;, &apos;workingEnvironmentType&apos;: &apos;VSA&apos;, &apos;creatorUserEmail&apos;: &apos;samlp|NetAppSAML|testuser&apos;}], &apos;onPremWorkingEnvironments&apos;: [], &apos;gcpVsaWorkingEnvironments&apos;: [], &apos;azureVsaWorkingEnvironments&apos;: [{&apos;publicId&apos;: &apos;VsaWorkingEnvironment-az123456&apos;, &apos;name&apos;: &apos;testazure&apos;, &apos;tenantId&apos;: &apos;Tenant-2345&apos;, &apos;isHA&apos;: False, &apos;cloudProviderName&apos;: &apos;Azure&apos;, &apos;workingEnvironmentType&apos;: &apos;VSA&apos;, &apos;creatorUserEmail&apos;: &apos;samlp|NetAppSAML|testuser&apos;}]}, None)}, &apos;changed&apos;: False},)
</system-out></testcase><testcase classname="modules.test_na_cloudmanager_nss_account.TestMyModule" file="modules/test_na_cloudmanager_nss_account.py" line="97" name="test_create_nss_account_idempotency" time="0.00279092788696"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_nss_account.py:97: &lt;py._xmlgen.raw object at 0x562ce4c97f10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_nss_account.TestMyModule" file="modules/test_na_cloudmanager_nss_account.py" line="118" name="test_create_nss_account_successfully" time="0.00341701507568"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_nss_account.py:118: &lt;py._xmlgen.raw object at 0x562ce4c97790&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_snapmirror.TestMyModule" file="modules/test_na_cloudmanager_snapmirror.py" line="109" name="test_module_fail_when_required_args_present" time="0.00239610671997"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_snapmirror.py:109: &lt;py._xmlgen.raw object at 0x562ce4c84e10&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_snapmirror.TestMyModule" file="modules/test_na_cloudmanager_snapmirror.py" line="119" name="test_create_cloudmanager_snapmirror_create_pass" time="0.00252795219421"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_snapmirror.py:119: &lt;py._xmlgen.raw object at 0x562ce4c64dd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_snapmirror.TestMyModule" file="modules/test_na_cloudmanager_snapmirror.py" line="154" name="test_delete_cloudmanager_snapmirror_delete_pass" time="0.00240087509155"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_snapmirror.py:154: &lt;py._xmlgen.raw object at 0x562ce4c7c250&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="96" name="test_module_fail_when_required_args_missing" time="0.00254106521606"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:96: &lt;py._xmlgen.raw object at 0x562ce4c8b6d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="103" name="test_create_volume_successfully" time="0.00274801254272"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:103: &lt;py._xmlgen.raw object at 0x562ce4c8bcd0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="120" name="test_create_volume_idempotency" time="0.00253820419312"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:120: &lt;py._xmlgen.raw object at 0x7feda00624d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="141" name="test_update_volume_successfully" time="0.00229811668396"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:141: &lt;py._xmlgen.raw object at 0x7feda00620d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="165" name="test_update_volume_idempotency" time="0.00223278999329"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:165: &lt;py._xmlgen.raw object at 0x7feda00629d0&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_info" file="modules/test_na_cloudmanager_info.py" line="158" name="test_get_aggregates_info" time="15.0597822666"><system-out>Info: test_create_cloudmanager_info: AnsibleExitJson({&apos;info&apos;: {&apos;working_environments_info&apos;: (None, &quot;HTTPSConnectionPool(host=&apos;cloudmanager.cloud.netapp.com&apos;, port=443): Max retries exceeded with url: /occm/api/working-environments (Caused by NewConnectionError(&apos;&lt;urllib3.connection.VerifiedHTTPSConnection object at 0x55a184e10e10&gt;: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution&apos;,))&quot;)}, &apos;changed&apos;: False},)
</system-out><system-err>No handlers could be found for logger &quot;ansible_collections.netapp.cloudmanager.plugins.module_utils.netapp&quot;
</system-err></testcase><testcase classname="modules.test_na_cloudmanager_volume.TestMyModule" file="modules/test_na_cloudmanager_volume.py" line="188" name="test_create_volume_by_workingenv_name_successfully" time="0.0027027130127"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_volume.py:188: &lt;py._xmlgen.raw object at 0x7feda006c210&gt;</skipped></testcase><testcase classname="modules.test_na_cloudmanager_snapmirror.TestMyModule" file="modules/test_na_cloudmanager_snapmirror.py" line="102" name="test_module_fail_when_required_args_missing" time="0.00337719917297"><skipped message="Skipping Unit Tests on 2.6 as unittest2 may not be available" type="pytest.skip">modules/test_na_cloudmanager_snapmirror.py:102: &lt;py._xmlgen.raw object at 0x7feda006c410&gt;</skipped></testcase></testsuite>